### PR TITLE
Make deployer input-model configurable (prep steps)

### PR DESCRIPTION
**Pull request by dirkmueller**
This isn't working, but adds the parameters to the JJB
template so that we can start implementation.

### CODE REVIEW

File: scripts/jenkins/ardana/heat-ardana-std-min.yaml [hunk 1]
```diff
@@ -0,0 +1,207 @@
+heat_template_version: 2016-10-14
```
Comment (toabctl) [inline scripts/jenkins/ardana/heat-ardana-std-min.yaml]: The file looks like a almost identical copy of the template we already have. is it possible to extract the duplicated parts somehow?

Comment (dirkmueller) [reply scripts/jenkins/ardana/heat-ardana-std-min.yaml]: dunno yet, wanted to do that after I understood what is is going to look like

File: jenkins/ci.suse.de/ardana-job.yaml [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+    workspace: "ardana-job-$BUILD_ID"
     # the selected node needs the cloud credentials
     node: cloud-ardana-ci
     wrappers:
       - timestamps
+      - build-name:
+          name: '#${BUILD_NUMBER}: ${ENV,var="job_name"}/${ENV,var="model"}'
+
+      - timeout:
+          timeout: 60
```
Comment (toabctl) [inline jenkins/ci.suse.de/ardana-job.yaml]: Is 60 min enough?

Comment (dirkmueller) [reply jenkins/ci.suse.de/ardana-job.yaml]: happy to raise it when it becomes an issue

**Pull request merged**